forum/forum/footer.jsp

```diff
@@ -1,7 +1,8 @@
+<!-- INF16B Maximilian Gerlach -->
 <div class=footer>&copy; 2017 TODO
    | <a href=‘contact.jsp’>Kontakt</a> 
```

Comment:
Es sollten nur double quotes `"` oder single quotes `'` in HTML/CSS verwendet werden.